src/Balances.sol

```diff
@@ -26,16 +26,28 @@ contract Balances {
         return scheduledForTransfer.length;
     }
 
+    function participantIsReadyForTransfer (address participant) public view returns (bool) {
+        for (uint i = 0; i < readyForTransfer.length; i++) {
+            if (readyForTransfer[i] == participant) {
+                return true;
+            }
+        }
+        return false;
+    }
+
     function increaseParticipantBalance(
         address payable participant,
         uint amount
     ) internal {
         uint oldBalance = balances[participant];
         uint newBalance = oldBalance + amount;
         balances[participant] = newBalance;
+        if (newBalance <= minBalanceForTransfer) {
```

Comment:
FWIW, both the current and the proposed versions consider the participant as ready to receive rewards only after they earn more than the treshold. I.e. if the threshold for transfer is 0.1 FIL and my scheduled rewards are exactly 0.1 FIL, I am not ready for transfer.

I think that could be rather confusing? However, let's open a new pull request if we decide to change that behavior.

Reply:
It's quite unlikely to land exactly at the threshold, but you're right, it should be possible. Will follow up with a PR

Reply:
https://github.com/Meridian-IE/impact-evaluator/issues/82